frontend/src/app/containers/App.js

```diff
@@ -36,6 +36,10 @@ class App extends Component {
 
     const experimentsPath = 'experiments/';
 
+    // These are used to expose page state for Optimizely, testing, etc.
```

Comment:
How are these used by Optimizely, exactly? 

Wondering in part because these variables won't get set immediately on page load, because of the conditions that come before. Namely, experiment data has to be loaded, and then if the add-on is installed it will wait for the installed add-ons list to be received.

I see there's a `sendToOptimizely` call later... is that where these get picked up?